build-system/tasks/e2e/describes-e2e.js

```diff
@@ -160,21 +166,20 @@ function createDriver(browserName, args, deviceName) {
         width: DEFAULT_E2E_INITIAL_RECT.width,
         height: DEFAULT_E2E_INITIAL_RECT.height,
       });
-      builder = new Builder()
+      return new Builder()
         .forBrowser('firefox')
-        .setFirefoxOptions(firefoxOptions);
+        .setFirefoxOptions(firefoxOptions)
+        .build();
     case 'chrome':
       const chromeOptions = new chrome.Options(capabilities);
       chromeOptions.addArguments(args);
       if (deviceName) {
         chromeOptions.setMobileEmulation({deviceName});
       }
-      builder = new Builder()
-        .forBrowser('chrome')
-        .setChromeOptions(chromeOptions);
+      const driver = chrome.Driver.createSession(chromeOptions);
+      driver.onQuit = null;
```

Comment:
If this is a workaround, maybe add a comment explaining why this is done?

Reply:
Done.